### DISCUSSION TITLE
[SDK-JS] Allow to as HTTP Headers for captcha

### DISCUSSION THREAD

**Discussion by pierre-H** [directus/directus#2836] [Feature Requests]
Hello,

First thank you to the Directus team for this great app !
Very excited to discover the next version 😃 !

Sometimes you need to send a captcha token in multiple requests.
It would be great to have the possibility to do a `api.updateItem('collection', {...}, {headers: {X-SECURITY-CHECK: token}})` and then with an item.create/update:before hook to check the captcha.

Actually, we can't set an HTTP header.

What do you think about that ?

**Comment by rijkvanzanten** [comment]
Sounds like a great idea!

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
